h2o-docs/src/booklets/v2_2015/source/GLMBooklet.tex

```diff
@@ -1398,6 +1399,128 @@ \subsection{Model Statistics}
 \waterExampleInPython
 \lstinputlisting[style=python]{GLM_Vignette_code_examples/glm_accessors.py}
 
+\subsubsection{GLM Likelihood and AIC}
+
+During model training, simplified formulas of likelihood and AIC are used. After the model is built, the full formula is used to calculate the output of the full log likelihood and full AIC values.
+
+\textbf{Note:} The log likelihood value is not available in the cross-validation metrics. The AIC value is available and is calculated using the original simplified formula independent of the log likelihood.
+
+The following are the supported GLM families and formulae (the log likelihood is calculated for the  \emph{i}th observation).
+
+\textbf{Gaussian:}
+
+$$l(\mu_i (\beta); y_i, w_i) = - \frac{1}{2} \Big[ \frac{w_i (y_i - \mu_i)^2}{\phi} + \log \big(\frac{\phi}{w_i} \big) + \log (2 \pi) \Big]$$
+
+where
+
+\begin{itemize}
+\item $\phi$ is the dispersion parameter estimation
+\item $\mu_i$ is the prediction
+\item $y_i$ is the real value of the target variable
+\end{itemize}
+
+\emph{Note:} You must estimate the dispersion parameter. During initialization, \texttt{compute\_p\_values} and \texttt{remove\_collinear\_columns)} must be set to \texttt{True} to facilitate this estimation process. Also, \texttt{dispersion\_parameter\_method} must be set to \texttt{"pearson"}.
+
+\textbf{Binomial:}
+
+$$l \big(\mu_i (\beta); y_i, w_i \big) = w_i \big(y_i \log \{ \mu_i \} + (1-y_i) \log \{ 1-\mu_i \} \big)$$
+
+where
+\begin{itemize}
+\item $\mu_i$ is the probability of 1
+\item $y_i$ is the real value of the target variable
+\end{itemize} 
+
+
+\textbf{Quasibinomial:}
+
+\begin{itemize}
+\item If the predicted value equals $y_i$, log likelihood is 0
+\item If $\mu_i >1$ then $l(\mu_i (\beta); y_i) = y_i \log \{ \mu_i \}$
+\item Otherwise, $l(\mu_i (\beta); y_i) = y_i \log \{ \mu_i \} + (1-y_i) \log \{ 1- \mu_i \}$ where
+\begin{itemize}
+\item $\mu_i$ is the probability of 1
+\item $y_i$ is the real value of the target variable
+\end{itemize}
+\end{itemize}
+
+\textbf{Fractional Binomial:}
+
+$$l(\mu_i (\beta); y_i) = w_i \Big(y_i \times \log \big(\frac{y_i}{\mu_i} \big) + (1-y_i) \times \log \big(\frac{1-y_i}{1-\mu_i} \big) \Big)$$
+
+where
+
+\begin{itemize}
+\item $\mu_i$ is the probability of 1
+\item $y_i$ is the real value of the target variable
+\end{itemize}
+
+\textbf{Poisson:}
+
+$$l(\mu_i (\beta); y_i) = w_i \big(y_i \times \log (\mu_i) - \mu_i - \log (\Gamma (y_i +1)) \big)$$
+
+where
+
+\begin{itemize}
+\item $\mu_i$ is the prediction
+\item $y_i$ is the real value of the target variable
+\end{itemize}
+
+\textbf{Negative Binomial:}
+
+$$l(\mu_i (\beta); y_i, w_i) = y_i \log \big(\frac{k \mu}{w_i} \big) - \big(y_i + \frac{w_i}{k} \big) \log \big(1 + \frac{k \mu}{w_i} \big) + \log \Big(\frac{\Gamma \big( y_i = \frac{w_i}{k} \big)} {\Gamma (y_i +1) \Gamma \big(\frac{w_i}{k}\big)} \Big)$$
```

Comment:
Please change

$$l(\mu_i (\beta); y_i, w_i) = y_i \log \big(\frac{k \mu}{w_i} \big) - \big(y_i + \frac{w_i}{k} \big) \log \big(1 + \frac{k \mu}{w_i} \big) + \log \Big(\frac{\Gamma \big( y_i = \frac{w_i}{k} \big)} {\Gamma (y_i +1) \Gamma \big(\frac{w_i}{k}\big)} \Big)$$

to

$$l(\mu_i (\beta); y_i, w_i) = y_i \log \big(\frac{k \mu}{w_i} \big) - \big(y_i + \frac{w_i}{k} \big) \log \big(1 + \frac{k \mu}{w_i} \big) + \log \Big(\frac{\Gamma \big( y_i + \frac{w_i}{k} \big)} {\Gamma (y_i +1) \Gamma \big(\frac{w_i}{k}\big)} \Big)$$